### PR TITLE
Integration Test debug changes. also added some CI doco and badge. template tweak

### PR DESCRIPTION
**Pull request by bhamail**
Sorry for the mixed bag of changes in here, but they're all pretty simple and harmless I think.

This pull request makes the following changes:
* Create "IT" workflow, to make it easier to try out IT's on CircleCI server to watch the fireworks.
* Add CI badge
* Remove GitHub issue template header

More to come later, but wanted to push these up first.

cc @bhamail / @DarthHater / @guillermo-varela

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,20 +1,3 @@
-<!--
```
Comment (guillermo-varela) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Why was this deleted?

Comment (guillermo-varela) [reply .github/ISSUE_TEMPLATE/bug_report.md]: > Why was this deleted?

Clarified with @DarthHater: "the header isn’t really needed on the file you asked about, that’s a bug template and it’s kinda unnecessary "

Comment (bhamail) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Also, the header shows up in the actual bug report - e.g. when you create a new bug report, BAM - the header text is shown. yuck.
